penguin/tools/core/skill_tools.py

```
@@ -0,0 +1,101 @@
+        payload["active"] = sorted(active_names)
+        payload["skills"] = [
+            {**skill, "active": skill.get("name") in active_names}
+            for skill in payload.get("skills", [])
+        ]
+        return json.dumps(payload, indent=2)
+
+    def activate_skill(
+        self,
+        name: str,
+        session_id: Optional[str] = None,
+        load_into_context: bool = True,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)